embed/Makefile.am

```diff
@@ -43,6 +43,10 @@ libephyembed_la_SOURCES = \
 	ephy-file-monitor.h		\
 	ephy-find-toolbar.c		\
 	ephy-find-toolbar.h		\
+	ephy-fx-password-notification.c	\
+	ephy-fx-password-notification.h	\
+	ephy-notification-manager.c	\
```

Comment:
Hm, now this is unfortunately a confusing name, because we also show HTML desktop notifications, which is what anyone looking at this file name will think of. Not sure how to avoid this. :/